app.js

```diff
@@ -1,8 +1,28 @@
 const express = require("express")
 const app = express()
 
+const { getAllTopics } = require("./controllers/controllers")
+
 app.use(express.json())
```

Comment:
Great to see that you are thinking ahead but we don't need this yet as we aren't expecting to handle any JSON bodies at this moment in time. We should only add code relevant to each ticket at each point/branch